common/unsafe/src/main/java/org/apache/spark/unsafe/array/ByteArrayMethods.java

```diff
@@ -52,11 +53,16 @@ public static int roundNumberOfBytesToNearestWord(int numBytes) {
    * Optimized byte array equality check for byte arrays.
    * @return true if the arrays are equal, false otherwise
    */
+  public static boolean arrayEqualsBlock(
```

Comment:
It should have new doc instead of using `arrayEquals`'s.

Reply:
yeah